### ISSUE TITLE
build: error with meson 0.60: "../meson.build:277:2: ERROR: Function accepts only strings, integers, lists, dictionaries and lists thereof."

### ISSUE THREAD

**Issue by vtorri** [ebassi/graphene#253]
graphene 1.10.8
meson 0.60
i am cross compiling (on windows, using msys2 and the mingw-w64 toolchain)

error:  ../meson.build:277:2: ERROR: Function accepts only strings, integers, lists, dictionaries and lists thereof.

no error with meson 0.55.3 (the minimal version requested in meson.build)

**Comment by ebassi** [comment]
It would be useful for debugging purposes if you checked what's at line 277 of the meson.build file.

Current tip of the main development branch has a `message()` function, which takes more than strings, integers, lists, and dictionaries: https://mesonbuild.com/Reference-manual_functions.html#message

Could you check what happens if the `message()` is replaced with:

```
  message('Cross-building, assuming IEEE 754 division: @0@'.format(ieee754_float_div))
```

**Comment by eli-schwartz** [comment]
This used to be undocumented and an accidental side effect of leaky python implementation details casting.

It's officially supported since https://github.com/mesonbuild/meson/commit/6298e2907082aff0763a09165c6c2841ac869e2d and was backported to 0.60.1

So, 

> graphene 1.10.8
> meson 0.60
> i am cross compiling (on windows, using msys2 and the mingw-w64 toolchain)

You're using something a lot more specific than Meson 0.60, you're using 0.60.0 and not 0.60.1, please specify this next time.

**Comment by vtorri** [comment]
> It would be useful for debugging purposes if you checked what's at line 277 of the meson.build file.

well, I specified the version of graphene

> Current tip of the main development branch has a `message()` function, which takes more than strings, integers, lists, and dictionaries: https://mesonbuild.com/Reference-manual_functions.html#message
> 
> Could you check what happens if the `message()` is replaced with:
> 
> ```
>   message('Cross-building, assuming IEEE 754 division: @0@'.format(ieee754_float_div))
> ```

works with meson  0.55.3, 0.60.0 and 0.60.1

output:

```
Message: Cross-building, assuming IEEE 754 division: true
```

**Comment by ebassi** [comment]
Since Meson has been fixed, and the 0.60.* branch is really old at this point, let's close this.
